vuepress/docs/.vuepress/components/SpecialLayout.vue

```diff
@@ -57,7 +57,7 @@
           <div style="display:none">
             <p>Download Entando custom resource definitions</p>
             <div class="instruction">
-              curl -L -C - https://dev.entando.org/custom-resources.tar.gz | tar -xz
+              curl -L -C - https://dev.entando.org/assets/yaml/custom-resources.tar.gz | tar -xz
```

Comment:
We need to work towards not hosting these here. This will eventually get out of sync and we'll suffer for it.

Reply:
Agreed. We may need a conversation around to do that but I added a ticket for it and linked it to the versioning story: https://jira.entando.org/browse/ENDOC-32